sync.cmd

```diff
@@ -25,22 +27,26 @@ if /I [%1]==[/?] goto Usage
 
 if /I [%1] == [/p] (
     set packages=true
-    set thisArgs=!thisArgs!%1
+    set thisArgs=%thisArgs%%1
     goto Next
 )
 
 if /I [%1] == [/s] (
     set src=true
-    set thisArgs=!thisArgs!%1
+    set thisArgs=%thisArgs%%1
     goto Next
 )
 
-if [!thisArgs!]==[] (
-  set unprocessedBuildArgs=!allargs!
-) else (
-  call set unprocessedBuildArgs=%%allargs:*!thisArgs!=%%
+if /I [%1] == [/t] (
```

Comment:
You can run without packagesdrops being set and it will work,  it will just grab package version information from the existing project.json and assembly information from the project reference. We will probably want a similar option for the test runs,  but this pr is only focused on the build portion,  @chrisboh would have more insight into the actual test run side. 

I wouldn't consider this a final version of sync.cmd. This was pulled from an early iteration of what @maririos was working on,  but she intends to make additional changes.  This change was useful to facilitate the work flow, but there is additional work to be done to align this with the other build  / clean / sync scripts.